src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,19 @@
 const accessibilityQuiz = [
+  {
+    Question:
+      "When creating charts, what should be added so that color is not the only identifier to convey the meaning of the information?",
+    Answer:
+      "Patterns.",
+    Distractor1:
+      "Hint Buttons.",
+    Distractor2:
+      "Gradients.",
```

Comment:
periods are not necessary here

```suggestion
    Answer:
      "Patterns",
    Distractor1:
      "Hint Buttons",
    Distractor2:
      "Gradients",
```

Reply:
Will change that, thanks!